vocab/credentials/v2/issuer-dependent.html

```diff
@@ -66,15 +68,58 @@
     <section>
         <h2>Issuer Dependent Terms</h2>
         <p>
-            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary items defined by the Verifiable Credentials Working Group through different vocabularies, such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. Applications using additional, user-dependent vocabularies of their own are encouraged to create their own formal vocabularies, and corresponding JSON-LD context definitions, to ensure the interoperability of the credentials they produce and/or consume.
+            The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> of a Verifiable Credential, 
+            as defined in [vc-data-model], provides a mapping from the standard terms used in credentials to the vocabulary 
+            items defined by the Verifiable Credentials Working Group through different vocabularies, 
+            such as the <a href="https://www.w3.org/2018/credentials/">Verifiable Credentials Vocabulary</a>. 
+            Applications using additional, user-dependent vocabularies of their own are encouraged to 
+            create their own formal vocabularies, and corresponding JSON-LD context definitions, 
+            to ensure the interoperability of the credentials they produce and/or consume.
         </p>
         <p>
-            There may be cases, however, when the creation of such formal vocabularies and context definitions is too much of a burden, such as for closed applications, experiments, or examples. The <a href="https://www.w3.org/TR/vc-data-model-2.0/#base-context">base context</a> has been constructed in such a way that issuers may use new, hitherto undefined terms without further ado, and these terms will be mapped onto URLs of the form <code>https://www.w3.org/ns/credentials/issuer-dependent#TERM</code>.
+            Not all users desire to define JSON-LD vocabularies or to make use of them with W3C Verifiable Credentials. 
```

Comment:
I changed this, because the previous content seems to be dismissive  / assume the intention of the issuer / holder.

I think we should assume their intention is to use the features we have built, which includes default expansion.

Reply:
-1, we shouldn't assume intention -- we don't know what it is. This landing place is specifically here because we have no idea why they're not defining their terms... all we know is that they didn't do it and so no one should be surprised when there is a term conflict in a large ecosystem.

Reply:
Let us not assume that the term will ever be defined or that the issuer made a mistake then.

Reply:
Outdating, resolving, please re review